pkg/agent/ssh/server.go

```diff
@@ -285,3 +324,62 @@ func (s *Server) sftpHandler(session ssh.Session) {
 		fmt.Println("sftp server completed with error:", err)
 	}
 }
+
+func (s *Server) getUserUidGid(username string) (int, int, []int, error) {
```

Comment:
I think this snippet might be enough for this entire function. Also, the snippet handles all OSs.

```
user, err := user.Lookup("user")
```

<img width="405" alt="Screenshot 2024-04-27 at 20 36 56" src="https://github.com/daytonaio/daytona/assets/26512078/b97f21d9-605e-4d8b-b395-db2283c8d722">

P.S. `user.HomeDir` can also be used so we don't "guess" with `/home/USER`.

Reply:
good point. ty.